website/docs/r/arc_kubernetes_cluster.html.markdown

```diff
@@ -0,0 +1,106 @@
+---
+subcategory: "ArcKubernetes"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_arc_kubernetes_cluster"
+description: |-
+  Manages an Arc Kubernetes Cluster.
+---
+
+# azurerm_arc_kubernetes_cluster
+
+Manages an Arc Kubernetes Cluster.
+
+-> **Note:** To connect an existing Kubernetes cluster to Azure Arc, besides creating an Arc Kubernetes Cluster in Azure, Azure Arc agents are also needed to deploy in the Kubernetes cluster. For more details refer to [Deploy agents to your cluster](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/conceptual-agent-overview#deploy-agents-to-your-cluster).
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_arc_kubernetes_cluster" "example" {
+  name                         = "example-akcc"
+  resource_group_name          = azurerm_resource_group.example.name
+  location                     = "West Europe"
+  agent_public_key_certificate = filebase64("testdata/public.cer")
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  tags = {
+    ENV = "Test"
+  }
+}
+```
+
```

Comment:
It might be helpful to add a note to point users to the example, something like
```suggestion

-> **Note:** An extensive example on connecting the `azurerm_arc_kubernetes_cluster` to an external kubernetes cluster can be found in [the `./examples/arckubernetes` directory within the GitHub Repository](https://github.com/hashicorp/terraform-provider-azurerm/tree/main/examples/arckubernetes).
```